<!DOCTYPE html><html lang="en-uk" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  
  
  <meta name="generator" content="Wowchemy 4.8.0 for Hugo">
  

  

  
  

  
  
  
  
  
    
    
    
  
  

  

  
  
  
    
  
  <meta name="description" content="Background&lt;br/&gt; The introduction of DNA-based molecular markers made a revolution in biological systematics. However, in cases of very recent divergence events, the neutral divergence may be too slow, and the analysis of adaptive part of the genome is more informative to reconstruct the recent evolutionary history of young species. The advantage of proteomics is its ability to reflect the biochemical machinery of life. It may help both to identify rapidly evolving genes and to interpret their functions. &lt;br/&gt; Methods&lt;br/&gt; Here we applied a comparative gel-based proteomic analysis to several species from the gastropod family Littorinidae. Proteomes were clustered to assess differences related to species, geographic location, sex and body part, using data on presence/absence of proteins in samples and data on protein occurrence frequency in samples of different species. Cluster support was assessed using multiscale bootstrap resampling and the stability of clustering—using cluster-wise index of cluster stability. Taxon-specific protein markers were derived using IndVal method. Proteomic trees were compared to consensus phylogenetic tree (based on neutral genetic markers) using estimates of the Robinson–Foulds distance, the Fowlkes–Mallows index and cophenetic correlation. &lt;br/&gt; Results&lt;br/&gt; Overall, the DNA-based phylogenetic tree and the proteomic similarity tree had consistent topologies. Further, we observed some interesting deviations of the proteomic littorinid tree from the neutral expectations. (1) There were signs of molecular parallelism in two Littoraria species that phylogenetically are quite distant, but live in similar habitats. (2) Proteome divergence was unexpectedly high between very closely related Littorina fabalis and L. obtusata, possibly reflecting their ecology-driven divergence. (3) Conservative house-keeping proteins were usually identified as markers for cryptic species groups (“saxatilis” and “obtusata” groups in the Littorina genus) and for genera (Littoraria and Echinolittorina species pairs), while metabolic enzymes and stress-related proteins (both potentially adaptively important) were often identified as markers supporting species branches. (4) In all five Littorina species British populations were separated from the European mainland populations, possibly reflecting their recent phylogeographic history. Altogether our study shows that proteomic data, when interpreted in the context of DNA-based phylogeny, can bring additional information on the evolutionary history of species.">

  
  <link rel="alternate" hreflang="en-uk" href="https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/">

  







  




  
  
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  

  
  
  
  <meta name="theme-color" content="#518f8f">
  

  
  
  
  <script src="/js/mathjax-config.js"></script>
  

  
  
  
  
    
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js" integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.css">

  





<script async src="https://www.googletagmanager.com/gtag/js?id=UA-131520217-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'UA-131520217-1', { 'anonymize_ip': true });

  
  document.addEventListener('click', onClickCallback, false);
</script>


  


  
  

  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="Marina Varfolomeeva">
  <meta property="og:url" content="https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/">
  <meta property="og:title" content="Proteomic similarity of the Littorinid snails in the evolutionary context | Marina Varfolomeeva">
  <meta property="og:description" content="Background&lt;br/&gt; The introduction of DNA-based molecular markers made a revolution in biological systematics. However, in cases of very recent divergence events, the neutral divergence may be too slow, and the analysis of adaptive part of the genome is more informative to reconstruct the recent evolutionary history of young species. The advantage of proteomics is its ability to reflect the biochemical machinery of life. It may help both to identify rapidly evolving genes and to interpret their functions. &lt;br/&gt; Methods&lt;br/&gt; Here we applied a comparative gel-based proteomic analysis to several species from the gastropod family Littorinidae. Proteomes were clustered to assess differences related to species, geographic location, sex and body part, using data on presence/absence of proteins in samples and data on protein occurrence frequency in samples of different species. Cluster support was assessed using multiscale bootstrap resampling and the stability of clustering—using cluster-wise index of cluster stability. Taxon-specific protein markers were derived using IndVal method. Proteomic trees were compared to consensus phylogenetic tree (based on neutral genetic markers) using estimates of the Robinson–Foulds distance, the Fowlkes–Mallows index and cophenetic correlation. &lt;br/&gt; Results&lt;br/&gt; Overall, the DNA-based phylogenetic tree and the proteomic similarity tree had consistent topologies. Further, we observed some interesting deviations of the proteomic littorinid tree from the neutral expectations. (1) There were signs of molecular parallelism in two Littoraria species that phylogenetically are quite distant, but live in similar habitats. (2) Proteome divergence was unexpectedly high between very closely related Littorina fabalis and L. obtusata, possibly reflecting their ecology-driven divergence. (3) Conservative house-keeping proteins were usually identified as markers for cryptic species groups (“saxatilis” and “obtusata” groups in the Littorina genus) and for genera (Littoraria and Echinolittorina species pairs), while metabolic enzymes and stress-related proteins (both potentially adaptively important) were often identified as markers supporting species branches. (4) In all five Littorina species British populations were separated from the European mainland populations, possibly reflecting their recent phylogeographic history. Altogether our study shows that proteomic data, when interpreted in the context of DNA-based phylogeny, can bring additional information on the evolutionary history of species."><meta property="og:image" content="https://varmara.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://varmara.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-uk">
  
    
      <meta property="article:published_time" content="2020-01-01T00:00:00&#43;00:00">
    
    <meta property="article:modified_time" content="2020-01-01T00:00:00&#43;00:00">
  

  


    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/"
  },
  "headline": "Proteomic similarity of the Littorinid snails in the evolutionary context",
  
  "datePublished": "2020-01-01T00:00:00Z",
  "dateModified": "2020-01-01T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "Arina L. Maltseva"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "Marina Varfolomeeva",
    "logo": {
      "@type": "ImageObject",
      "url": "https://varmara.github.io/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png"
    }
  },
  "description": "Background\u003cbr/\u003e The introduction of DNA-based molecular markers made a revolution in biological systematics. However, in cases of very recent divergence events, the neutral divergence may be too slow, and the analysis of adaptive part of the genome is more informative to reconstruct the recent evolutionary history of young species. The advantage of proteomics is its ability to reflect the biochemical machinery of life. It may help both to identify rapidly evolving genes and to interpret their functions. \u003cbr/\u003e Methods\u003cbr/\u003e Here we applied a comparative gel-based proteomic analysis to several species from the gastropod family Littorinidae. Proteomes were clustered to assess differences related to species, geographic location, sex and body part, using data on presence/absence of proteins in samples and data on protein occurrence frequency in samples of different species. Cluster support was assessed using multiscale bootstrap resampling and the stability of clustering—using cluster-wise index of cluster stability. Taxon-specific protein markers were derived using IndVal method. Proteomic trees were compared to consensus phylogenetic tree (based on neutral genetic markers) using estimates of the Robinson–Foulds distance, the Fowlkes–Mallows index and cophenetic correlation. \u003cbr/\u003e Results\u003cbr/\u003e Overall, the DNA-based phylogenetic tree and the proteomic similarity tree had consistent topologies. Further, we observed some interesting deviations of the proteomic littorinid tree from the neutral expectations. (1) There were signs of molecular parallelism in two Littoraria species that phylogenetically are quite distant, but live in similar habitats. (2) Proteome divergence was unexpectedly high between very closely related Littorina fabalis and L. obtusata, possibly reflecting their ecology-driven divergence. (3) Conservative house-keeping proteins were usually identified as markers for cryptic species groups (“saxatilis” and “obtusata” groups in the Littorina genus) and for genera (Littoraria and Echinolittorina species pairs), while metabolic enzymes and stress-related proteins (both potentially adaptively important) were often identified as markers supporting species branches. (4) In all five Littorina species British populations were separated from the European mainland populations, possibly reflecting their recent phylogeographic history. Altogether our study shows that proteomic data, when interpreted in the context of DNA-based phylogeny, can bring additional information on the evolutionary history of species."
}
</script>

  

  


  
  
  
  
  
    <script src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.js" integrity="sha256-5VhCqFam2Cn+yjw61zbBNrbHVJ6SRydPeKopYlngbiQ=" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.css" integrity="sha256-zQ0LblD/Af8vOppw18+2anxsuaz3pWYyVWi+bTvTH8Q=" crossorigin="anonymous">
  
  <script>
  window.addEventListener("load", function(){
    window.cookieconsent.initialise({
      "palette": {
        "popup": {
          "background": "#518f8f",
          "text": "rgb(255, 255, 255)"
        },
        "button": {
          "background": "rgb(255, 255, 255)",
          "text": "#518f8f"
        }
      },
      "theme": "classic",
      "content": {
        "message": "This website uses cookies to ensure you get the best experience on our website.",
        "dismiss": "Got it!",
        "link": "Learn more",
        "href": "/privacy/"
      }
    })});
  </script>



  





  <title>Proteomic similarity of the Littorinid snails in the evolutionary context | Marina Varfolomeeva</title>

</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class=" ">

  
  
  
    <script>window.wcDarkLightEnabled = true;</script>
  
  
    <script>const isSiteThemeDark = false;</script>
  
  
  <script src="/js/load-theme.js"></script>

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  












<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">Marina Varfolomeeva</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">Marina Varfolomeeva</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>About</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span>Blog</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#teaching"><span>Teaching</span></a>
        </li>

        
        

        

        
        
        
          
        

        

        
        
        
        

        
          
            
            
          
          
            
            
            
              
            
            
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-moon" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>



  <div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Proteomic similarity of the Littorinid snails in the evolutionary context</h1>

  

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Arina L. Maltseva</span>, <span >Marina A. Varfolomeeva</span>, <span >Arseniy A. Lobov</span>, <span >Polina Tikanova</span>, <span >Marina Panova</span>, <span >Natalia Mikhailova</span>, <span >Andrei I. Granovitch</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2020
  </span>
  

  

  

  
  
  

  
  

</div>

    











  



<div class="btn-links mb-3">
  
  








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 js-cite-modal"
        data-filename="/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1" href="https://doi.org/10.7717/peerj.8546" target="_blank" rel="noopener">
  DOI
</a>



</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">Background<br/> The introduction of DNA-based molecular markers made a revolution in biological systematics. However, in cases of very recent divergence events, the neutral divergence may be too slow, and the analysis of adaptive part of the genome is more informative to reconstruct the recent evolutionary history of young species. The advantage of proteomics is its ability to reflect the biochemical machinery of life. It may help both to identify rapidly evolving genes and to interpret their functions. <br/> Methods<br/> Here we applied a comparative gel-based proteomic analysis to several species from the gastropod family Littorinidae. Proteomes were clustered to assess differences related to species, geographic location, sex and body part, using data on presence/absence of proteins in samples and data on protein occurrence frequency in samples of different species. Cluster support was assessed using multiscale bootstrap resampling and the stability of clustering—using cluster-wise index of cluster stability. Taxon-specific protein markers were derived using IndVal method. Proteomic trees were compared to consensus phylogenetic tree (based on neutral genetic markers) using estimates of the Robinson–Foulds distance, the Fowlkes–Mallows index and cophenetic correlation. <br/> Results<br/> Overall, the DNA-based phylogenetic tree and the proteomic similarity tree had consistent topologies. Further, we observed some interesting deviations of the proteomic littorinid tree from the neutral expectations. (1) There were signs of molecular parallelism in two Littoraria species that phylogenetically are quite distant, but live in similar habitats. (2) Proteome divergence was unexpectedly high between very closely related Littorina fabalis and L. obtusata, possibly reflecting their ecology-driven divergence. (3) Conservative house-keeping proteins were usually identified as markers for cryptic species groups (“saxatilis” and “obtusata” groups in the Littorina genus) and for genera (Littoraria and Echinolittorina species pairs), while metabolic enzymes and stress-related proteins (both potentially adaptively important) were often identified as markers supporting species branches. (4) In all five Littorina species British populations were separated from the European mainland populations, possibly reflecting their recent phylogeographic history. Altogether our study shows that proteomic data, when interpreted in the context of DNA-based phylogeny, can bring additional information on the evolutionary history of species.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            
            <a href="/publication/#2">
              Journal article
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9">In: PeerJ, (8), <em>pp. e8546</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"></div>

    






<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/&amp;text=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context" target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/&amp;t=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context" target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context&amp;body=https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/&amp;title=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context" target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="whatsapp://send?text=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context%20https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=https://varmara.github.io/publication/2020-proteomic-similarity-of-the-littorinid-snails-in-the-evolutionary-context/&amp;title=Proteomic%20similarity%20of%20the%20Littorinid%20snails%20in%20the%20evolutionary%20context" target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>











  
  
    




  
    




  
    




  
    




  
    




  
    




  
    




  












  
  
  <div class="article-widget content-widget-hr">
    <h3>Related</h3>
    <ul>
      
      <li><a href="/publication/2019-colonies-as-dynamic-systems_-reconstructing-the-life-history-of-cribrilina-annulata/">Colonies as dynamic systems: reconstructing the life history of Cribrilina annulata (Bryozoa) on two algal substrates</a></li>
      
      <li><a href="/publication/2012-intraspecific-variation-in-stable-isotope-signatures-indicates-no-small-scale-feeding-interference/">Intraspecific variation in stable isotope signatures indicates no small-scale feeding interference between a horse mussel and an ascidian</a></li>
      
      <li><a href="/publication/2008-multiple-foundation-species-shape-benthic-habitat-islands/">Multiple foundation species shape benthic habitat islands</a></li>
      
      <li><a href="/publication/2007-effect-of-habitat-architecture-on-mibile-benthic-macrophauna/">Effect of habitat architecture on mobile benthic macrofauna associated with patches of barnacles and ascidians</a></li>
      
    </ul>
  </div>
  





  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js" integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js"></script>
        
      

    

    
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/wowchemy.min.4c2bca31150ce93c5a5e43b8a50f22fd.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  
  <p class="powered-by">
    
      <a href="/privacy/">Privacy Policy</a>
    
    
  </p>
  

  <p class="powered-by">
    © Marina A. Varfolomeeva 2018-2021
  </p>

  
  





  
  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://search.creativecommons.org/static/img/cc_icon.svg" alt="CC icon">
      <img src="https://search.creativecommons.org/static/img/cc-by_icon.svg" alt="CC by icon">
      
        <img src="https://search.creativecommons.org/static/img/cc-nc_icon.svg" alt="CC NC icon">
      
      
        <img src="https://search.creativecommons.org/static/img/cc-sa_icon.svg" alt="CC SA icon">
      
    </a>
  </p>




  <p class="powered-by">
    
    Published with
    <a href="https://wowchemy.com" target="_blank" rel="noopener">Wowchemy</a>  —
    the free, <a href="https://github.com/wowchemy/wowchemy-hugo-modules" target="_blank" rel="noopener">
    open source</a> website builder that empowers creators.
    

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
